python/dcr_template.py

```diff
@@ -780,18 +806,37 @@ def build_regularization(x_size=None, y_size=None, n_step=None, spatial_regulari
             reg_lambda = np.zeros((n_step*x_size*y_size, (n_step - 1)*x_size*y_size))
             for f in range(n_step - 1):
                 for ij in range(x_size*y_size):
-                    reg_lambda[f*x_size*y_size + ij, f*x_size*y_size + ij] = 1
-                    reg_lambda[(f + 1)*x_size*y_size + ij, f*x_size*y_size + ij] = -1
-            reg_lambda = np.append(reg_lambda, -reg_lambda, axis=1)
+                    reg_lambda[f*x_size*y_size + ij, f*x_size*y_size + ij] = 2*weight
+                    reg_lambda[(f + 1)*x_size*y_size + ij, f*x_size*y_size + ij] = -2*weight
+            # Use 2*weight above instead of extending with np.append(reg_lambda, -reg_lambda, axis=1)
```

Comment:
It's an explanation of a choice made in the preceding lines of code. I'll expand the comment to make my intention more clear.